features/main/hud/src/main/java/com/vgleadsheets/features/main/hud/HudFragment.kt

```diff
@@ -388,12 +390,22 @@ class HudFragment : VglsFragment() {
             partsExpanded
         )
 
-        val songDisplayClickHandler = {
+        val songDetailClickHandler = {
             if (currentSong != null) {
                 getFragmentRouter().showSheetDetail(currentSong.id)
             }
         }
 
+        val youtubeSearchClickHandler = {
+            if (currentSong != null) {
+                val query = "${currentSong.gameName} - ${currentSong.name}"
```

Comment:
Could you please add " music" on the end of the query string? The website has been using that for a while and it seems to produce better results.